actionmailer/lib/action_mailer/base.rb

```diff
@@ -486,7 +486,7 @@ class Base < AbstractController::Base
 
     helper ActionMailer::MailHelper
 
-    class_attribute :delivery_job, default: ::ActionMailer::MailDeliveryJob
+    class_attribute :delivery_job, default: false
```

Comment:
```suggestion
    class_attribute :delivery_job, default: nil
```

I think `nil` makes more sense than `false` here

I see that `destroy_association_async_job` has `false` for its default, but I'm not sure why

Reply:
I changed the defaults for both `ActionMailer::Base.delivery_job` and `ActiveRecord::Base.destroy_association_async_job` to be `nil`. 

I did a sweep through the code and didn't see any deliberate `nil?` or `== false` checks to differentiate between `nil` or `false`.